vimdoc/module.py

```diff
@@ -60,11 +60,17 @@ def Merge(self, block, namespace=None):
       # Overwrite existing section if it's a default.
       if block_id not in self.sections or self.sections[block_id].IsDefault():
```

Comment:
I'm surprised we _are_ trying to overwrite non-defaults with defaults, but if that's what we need to do, then this is certainly reasonable.